assets/scss/pages/cart.scss

```diff
@@ -0,0 +1,5 @@
+.p-cart__totals {
```

Comment:
Why it's here? You have `components/totals` which seems like a perfect place for it